command_line/refine.py

```diff
@@ -383,6 +384,13 @@ def run(args=None, phil=working_phil):
             "tasks.\n"
         )
 
+    if params.refinement.parameterisation.scan_varying is not False:
+        # duplicate crystal if necessary for scan varying
+        if len(experiments) > len(experiments.crystals()):
+            logger.info("Splitting crystal models before refinement")
```

Comment:
I don't think this message is very meaningful. What is 'splitting a crystal model'? I would prefer a more verbose but explicit description like `Ensuring a separate crystal model exists for each scan in refinement` or similar.

Reply:
Literally hold no opinion here 🙂`dials.refine` output is entirely your call

Reply:
Made more explicit:

```
Separating crystal model for experiment 1
Separating crystal model for experiment 2
Separating crystal model for experiment 3
```

Reply:
I still think `Separating crystal model` is not any more meaningful than `Splitting crystal model` to the user because it does not explain _why_ the action is taken.

Reply:
I would settle with `Copying crystal model for scan-varying refinement of experiment N`, which at least hints at why, and accurately describes what happens (a copy) rather than 'separating' which is ambiguous.

Reply:
_Duplicating_ crystal model...?

Reply:
Please make it as verbose as you like - personally I think no user will ever read it anyway but I recognise your territory here @dagewa